subscription-service/src/main/resources/db/migration/V0_19__expand_size_entity_info_type_solomn.sql

```diff
@@ -0,0 +1 @@
+alter table entity_info alter COLUMN "type" type varchar(1024);
```

Comment:
what is this doing here?

Reply:
I update the size of the column because for some test the size is too small currently (128)